src/t_hash.c

```diff
@@ -793,61 +874,16 @@ static void hashTypeRandomElement(robj *hashobj, unsigned long hashsize, listpac
  * Hash type commands
  *----------------------------------------------------------------------------*/
 
-void hsetnxCommand(client *c) {
-    robj *o;
-    if ((o = hashTypeLookupWriteOrCreate(c, c->argv[1])) == NULL) return;
-
-    if (hashTypeExists(o, c->argv[2]->ptr)) {
-        addReply(c, shared.czero);
-    } else {
-        hashTypeTryConversion(o, c->argv, 2, 3);
-        hashTypeSet(o, c->argv[2]->ptr, c->argv[3]->ptr, HASH_SET_COPY);
-        signalModifiedKey(c, c->db, c->argv[1]);
-        notifyKeyspaceEvent(NOTIFY_HASH, "hset", c->argv[1], c->db->id);
-        server.dirty++;
-        addReply(c, shared.cone);
-    }
-}
-
-void hsetCommand(client *c) {
-    int i, created = 0;
-    robj *o;
-
-    if ((c->argc % 2) == 1) {
-        addReplyErrorArity(c);
-        return;
-    }
-
-    if ((o = hashTypeLookupWriteOrCreate(c, c->argv[1])) == NULL) return;
-    hashTypeTryConversion(o, c->argv, 2, c->argc - 1);
-
-    for (i = 2; i < c->argc; i += 2) created += !hashTypeSet(o, c->argv[i]->ptr, c->argv[i + 1]->ptr, HASH_SET_COPY);
-
-    signalModifiedKey(c, c->db, c->argv[1]);
-    notifyKeyspaceEvent(NOTIFY_HASH, "hset", c->argv[1], c->db->id);
-    server.dirty += (c->argc - 2) / 2;
-
-    /* HMSET (deprecated) and HSET return value is different. */
-    char *cmdname = c->argv[0]->ptr;
-    if (cmdname[1] == 's' || cmdname[1] == 'S') {
```

Comment:
Not related to this PR, but I noticed it while we were talking.

Doesn't his break if you rename HSET?

Reply:
@madolson I guess so.